andela/kingsmen-ah-frontend
# #167221908 Deploy Base(staging) Application To Heroku

#### What does this PR do?
Set Up Heroku Deploy

#### Description of Task to be completed?
- Configure repo static and app json files
- Create kingsmen-ah-frontend pipeline via heroku
- Create kingsmen-ah-frontend-staging application
- Enable review apps

#### How should this be manually tested?
- Visit https://kingsmen-ah-frontend-staging.herokuapp.com/ to see the deployed application

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167221908](https://www.pivotaltracker.com/story/show/167221908)

#### Screenshots (if appropriate)
<img width="1342" alt="Screenshot 2019-07-12 at 7 25 25 AM" src="https://user-images.githubusercontent.com/22410701/61106841-3f809b80-a476-11e9-9816-a6b681a21e4e.png">

#### Questions:
N/A
